data.js

```diff
@@ -103,16 +103,36 @@ exports.data = [
             label: "greenpeace",
         },
         description: {
-            content: "Greenpeace is another big player. They also do work on a big scale as you can see on the Income they have to put them into their projects. Greenpeace has a lot of Actions going on.",
-            updatedBy: "Andre", updatedAt: "2019-08-18"
+            content: "Greenpeace is another big player, working in independent national and regional organizations. They do work on a big scale as you can see on the income and the variety of actions.",
+            updatedBy: "Andre", updatedAt: "2020-01-22"
         },
         established: {
             content: "[1971](http://www.tiki-toki.com/timeline/entry/594418/Greenpeace/#vars!date=1942-02-09_13:52:01!)",
             updatedBy: "Andre", updatedAt: "2019-08-06"
         },
         kpis: {
-            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions",
-            updatedBy: "Andre", updatedAt: "2019-08-06",
+            content: "- Actions done / pa \n- Participants in actions\n- Members\n- Press Mentions \n- Website visitors",
```

Comment:
1. I don't think "Website visitors"  is a KPI for Greenpeace
2. casing is inconsistent (Mentions, actions)

Reply:
done